src/tests.rs

```diff
@@ -223,9 +220,7 @@ fn test_rust_function() {
     ).unwrap();
 
     let lua_function = globals.get::<_, Function>("lua_function").unwrap();
-    let rust_function = lua.create_function(|lua, _| {
-        lua.pack("hello")
-    });
+    let rust_function = lua.create_function(|_, _: ()| Ok("hello"));
```

Comment:
Yeah, most places in Rust that accept an identifier actually work with irrefutable patterns. This includes `let (x, y) = ...`, closure arguments, but also function arguments so you can do something like `fn move(&mut self, (x, y): (i32, i32))` in case you didn't know